### PR TITLE
Add human and demihuman organs

### LINKED ISSUE #56100: NPCs and ferals are lacking a brain and many more organs
### Describe the bug

The only organ you get from butchering human corpses is the stomach
There's no other organ at all and it feels wierd that other humans are literaly brainless

### Steps to reproduce

Spawn and kill some NPCs and ferals
Butcher/dissect them
Notice the absence of most organs

### Expected behavior

Non-mutant humans have normal organs

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22000.556 (21H2)
- Game Version: 20b4ebb [32-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by AtomicFox556**
#### Summary
Content "Add human and demihuman organs"

#### Purpose of change
Fix #56100

#### Describe the solution
Add human and demihuman variants of meat scrap, liver, kidney, brain, sweetbread and lung, obtained by butchering human/demihuman corpses, and add them to all the recipes. Also adjusted amount of products harvested from such corpses a bit, for better consistency.

#### Describe alternatives you've considered
None.

#### Testing
Made sure that game runs with no errors and that cannibalism-related flags are still passed over when cooking.

#### Additional context
None.

### CODE REVIEW

File: data/json/items/comestibles/carnivore.json [hunk 1]
```diff
@@ -833,6 +923,24 @@
...
+    "description": "Thymus or pancreas from a human, which appears to have been extracted too crudely to ever be useful as a transplant.",
+    "material": [ "hflesh" ]
+  },
+  {
+    "id": "demihuman_sweetbread",
+    "type": "COMESTIBLE",
+    "copy-from": "sweetbread",
+    "looks_like": "sweetbread",
+    "name": { "str": "raw demihuman sweetbread" },
+    "description": "Thymus or pancreas extracted from a demihuman being.",
```
Comment (Maleclypse) [inline data/json/items/comestibles/carnivore.json:941]:
```suggestion
    "description": "The consistency of this organ suggests possibly a thymus or pancreas equivalent.",
```

File: data/json/items/comestibles/carnivore.json [hunk 2]
```diff
@@ -785,6 +857,24 @@
...
+    "description": "Kidney from a human, which appears to have been extracted too crudely to ever be useful as a transplant.",
+    "material": [ "hflesh" ]
+  },
+  {
+    "id": "demihuman_kidney",
+    "type": "COMESTIBLE",
+    "copy-from": "kidney",
+    "looks_like": "kidney",
+    "name": { "str": "raw demihuman kidney" },
+    "description": "Kidney extracted from a demihuman being.",
```
Comment (Maleclypse) [inline data/json/items/comestibles/carnivore.json:875]:
```suggestion
    "description": "This is probably a kidney extracted from a demihuman being.",
```

**Review by Maleclypse — COMMENTED** [review]
Added some ideas for different texts on a couple of the demihuman items.

File: data/json/items/comestibles/carnivore.json (hunk 2 above)
Comment (PatrikLundell) [reply data/json/items/comestibles/carnivore.json]: As far as I understand, people versed in butchering are able to identify organs from different species without too much trouble. Unless demihumans are supposed to come from another dimension, and thus won't be of an Earth evolutionary origin, identifying their organs ought to be fairly straightforward.

**Pull request merged**